### DISCUSSION TITLE
What is the schedule of community meetings?

### DISCUSSION THREAD

**Discussion by draghuram** [vmware-tanzu/velero#5800] [General discussions]
Hi,

I wanted to find out when the meetings occur but couldn't find the info from this page:
    https://velero.io/community/

Can someone let me know the schedule? 

Thanks,
Raghu

**Comment by Satsank** [comment]
1st and 3rd Tuesday at 12PM ET / 9AM PT (https://vmware.zoom.us/j/551441444?pwd=dHJyMWZtdHFPWWFJaTh5TnFuYWMvZz09)

2nd and 4th Tuesday at 8am China Standard Time / 8pm ET / 5pm PT (https://vmware.zoom.us/j/551441444?pwd=dHJyMWZtdHFPWWFJaTh5TnFuYWMvZz09)

**Reply by draghuram** [reply]
Thanks.

**Reply by sseago** [reply]
@Satsank Actually, that's out-of-date. The current meeting schedule is actually listed on the community page linked above:
```
Bi-weekly community meeting alternating every week between Beijing Friendly timezone and EST/Europe Friendly Timezone
Beijing/US friendly - we start at 8am Beijing Time(bound to CST) / 8pm EDT(7pm EST) / 5pm PDT(4pm PST) / 2am CEST(1am CET) 
US/Europe friendly - we start at 10am ET(bound to ET) / 7am PT / 3pm CET / 10pm(11pm) CST
```
Next week's meeting is the Beijing/US meeting, alternating every other week.

**Comment by draghuram** [comment]
Thansk @sseago but it is not clear which day the meetings are on (I am guessing they are on Tuesdays) unless I am missing something obvious. BTW, Is there any calendar invite for the calls?
